core/src/main/java/com/graphhopper/routing/VirtualEdgeIteratorState.java

```diff
@@ -215,6 +292,8 @@ public int getOrigEdgeLast() {
     @Override
     public EdgeIteratorState detach(boolean reverse) {
         if (reverse) {
+            // update properties of reverse edge
+            // TODO copy wayGeometry too
```

Comment:
Did you mean `pointList` with `wayGeometry` here ?

Reply:
yes (-> PointList fetchWayGeometry(int mode) and setWayGeometry(PointList list) )